repose-aggregator/core/repose-core-api/src/main/resources/META-INF/schema/container/container-configuration.xsd

```diff
@@ -252,6 +257,55 @@
         </xs:simpleContent>
     </xs:complexType>
 
+    <xs:complexType name="ViaConfiguration">
+        <xs:annotation>
+            <xs:documentation>
+                <html:p>
+                    Contains the information needed to configure Repose's Via header.
+                </html:p>
+            </xs:documentation>
+        </xs:annotation>
+        <xs:simpleContent>
+            <xs:extension base="papic:EmptyString">
+                <xs:attribute name="enabled" type="xs:boolean" use="optional" default="true">
+                    <xs:annotation>
+                        <xs:documentation>
+                            <html:p>Indicates whether or not the Via header is created.</html:p>
+                        </xs:documentation>
+                    </xs:annotation>
+                </xs:attribute>
+                <xs:attribute name="content" type="xs:string" use="optional">
```

Comment:
Got it. Man, I suck at names. I am not the one to ask. `prefix` or `value-prefix` seem legit. `content` just seems like it would be the entire value to me, especially alongside `enabled`.

It was my understanding that what is currently the `enabled` attribute would enable/disable the Repose _portion_ of the `Via` header, not the entire `Via` header. That way, the user has complete control over what ends up in the `Via` header, and whether or not the `Via` header is set at all. I also like that approach better. So I choose to accept the second part you have quoted and reject the first part. 😉 

Summoning @adrianjgeorge for final clarification.